spec/http.md

```diff
@@ -0,0 +1,51 @@
+HTTP
+====
+
+## Resources
+
+Each EC2 instance MUST present the following (GET) resources:
+
+    [/service/healthcheck](#healthcheck)
+    [/service/gtg](#gtg)
+    [/service/dependencies](#dependencies)
+
+These resources should be presented on port 8080 (not port 80).
```

Comment:
Definitely not port 80, as that implies root permissions, nginx, and all kinds of hassle.

Is this implicitly saying that the service should expose its HTTP API on port 8080 and the management interface should be on the same port as that? If so, I would explicitly state both things, but make both of them SHOULD rather than MUST.